docs/designs/conceptual-versioning.md

```diff
@@ -0,0 +1,520 @@
+
+    To better illustrate scenarios, in below sections, we call URL without `{host}` and `{docset-base-path}` as the **SitePath** of the file. In our example, **SitePath** for `articles/v1.0/a.md` and `articles/v2.0/a.md` are the same: `articles/a`.
+
+    If in one round of build, different files with the same **SitePath** are included, if there are two files without `monikerRange` or the intersection of any two files's *moniker list* is not empty, an error throws. For example, `articles/v1.0/a.md` has monikers `v1.0, v2.0` while `articles/v2.0/a.md` has monikers `v2.0, v3.0`, **an error throws** as for version `v2.0`, the result is indeterministic.
+
+4. `monikerDefinitionUrl` is an API, which provide the definition for moniker list, both *file* and *http(s)* URI schemas are supported. The moniker definition defines the *moniker name*, *product name*, *order* and other metadata for moniker.
+
+    A better user experience sample when using the new config:
+
+    If there is a file `articles/folder1/a.md` with monikerRange `'netcore-1.0'`, and file `articles/folder2/b.md` with monikerRange `'netcore-2.0'`, and you have to add a file `c.md` with monikerRange `'netcore-1.0 || netcore-2.0'`.
+
+    In docfx v2, since you cannot have overlap monikerRange in config, you have to copy `c.md` to `articles/folder1/` and `articles/folder2/`, which will bring you more maintenance cost for `c.md`.
+
+    But in docfx v3, you just need to maintenance one `c.md`
+
+##### 2.1.2 YAML Header
+
+For conceptual file, file level monikerRange setting is supported by using YAML header.
+
+```markdown
+---
+monikerRange: >=netcore-1.0
+---
+```
+
+##### 2.1.3 Moniker zone
+
+Inside the file, with help of [versioned zone syntax](https://review.docs.microsoft.com/en-us/new-hope/resources/conceptual-versioning?branch=master#configuring-markdown-files-with-versioned-zones), the file has ability to wrap versioned content into different zones.
+
+```markdown
+---
+monikerRange: >=netcore-1.0
+---
+
+# Moniker Zone sample
+
+content in `>=netcore-1.0`
+
+::: moniker range="netcore-1.0"
+content just in `netcore-1.0`
+::: moniker-end
+
+::: moniker range=">netcore-1.0"
+content just in `>netcore-1.0`
+::: moniker-end
+```
+
+> These three level moniker range config should follow these rules:
+> 1. The final file level moniker range is the intersection of moniker range from `config file` and moniker range from `YAML header`, if the intersection is empty, an error throws.
+> 2. The final zone moniker range is the intersection of moniker range of this file and moniker range of this zone. If the intersection is empty, an error throws.
+
+#### 2.2 Link/Xref
+
+In phase 1, content writer is allowed to reference another file with specific version by query string `?view={moniker}`, and the query string will be append to the resolve result, but we will not do the version existed check, and DHS will handle the fallback logic.
```

Comment:
>In phase 1 [](start = 0, length = 10)

can you also elaborate the other cases: reference a link without specific `view=` query string